src/test/java/LoginTests.java

```diff
@@ -1,10 +1,23 @@
+import PageObjectModel.HomePage;
+import PageObjectModel.LoginPage;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
 public class LoginTests extends BaseTest {
 
+    @Test
+    public void loginValidEmailPasswordTest()throws InterruptedException {
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+        loginPage.provideEmail(("demo@class.com"));
+        loginPage.providePassword(("te$t$tudent"));
+        loginPage.clickSubmit();
+
+        Assert.assertTrue(homePage.getUserAvatar().isDisplayed());
+    }
+
     @Test
     public void loginEmptyEmailPassword() {
```

Comment:
Hello,
Your homework 22 has been reviewd. I can see you refactor code and implement to use Page Object Model for LoginTests class. Great! But test script execute failed at `loginEmptyEmailPassword ` function test. Please use POM for this function and correct it. 

![image](https://github.com/eoparah/A51-QA-Automation-Framework/assets/144403954/33cf3a95-1dc3-4ad2-8ebd-be14ffa132a4)